smack-extensions/src/main/java/org/jivesoftware/smackx/blocking/BlockingCommandManager.java

```diff
@@ -163,14 +163,12 @@ public boolean isSupportedByServer()
     public List<Jid> getBlockList()
```

Comment:
@Flowdalic It is a refactor, look that:
`return Collections.unmodifiableList(blockListCached);`
is twice there without necessity